LuaUI/widgets/chobby/components/chat_windows.lua

```diff
@@ -153,6 +155,26 @@ function ChatWindows:init()
                 height = 40,
                 OnClick = { function() self:Minimize() end },
             },
+            Button:New {
+				width = 60,
+				y = 10,
+				right = 112,
+				height = 40,
+				caption = i18n("join"),
+				OnClick = { function()
+					if self.joinWindow == nil then
```

Comment:
you should set it to nil in places you dispose it (e.g. https://github.com/Spring-Chobby/Chobby/pull/18/files#diff-4620b7b610edd92d5b141cdf67d3cc28R398)